find.py

```diff
@@ -0,0 +1,21 @@
+import os
+from imports import *
+
+def find(field,search):
+    if field=="isbn":
+        list=db.session.query(Book).filter(Book.isbn.like(search))
```

Comment:
You will get errors with this implimentation, Run the application to check wether working add the required imports